web3/content_script.js

```diff
@@ -105,6 +105,7 @@ function main () {
         origin
       }, (result) => {
         if (!result) {
+          retry(ev);
```

Comment:
Why would there be no result ? If it's a network or overload issue, we might want to add a small timeout ?